File: Java/Searching/LinearSearch.java
package Java.Searching;
import java.util.Scanner;

public class LinearSearch
{
    public static void main(String args[])
    {
        Scanner scanner = new Scanner(System.in);
        int arr[]=new int[5];

        System.out.println("Enter 5 elements of the array");
   

        for(int i=0;i<5;i++)
        {
            arr[i]=scanner.nextInt();
        }
        System.out.println("Enter key elements to search");
        int key = scanner.nextInt();
        boolean flag=true;
        for(int i:arr)
        {
            if(i==key)
            {
                flag=false;
                System.out.println("Element Found");

            }
        }
        if(flag)
        {
            System.out.println("Element Not Found");

        }
        scanner.close();
    }
}

File: Python/Searching/linear_search.py
arr = input("enter elements of the array\n").strip(" ").split(" ")
key = input("\nenter key element to be found\n")
print("\ngiven array is arr: ",arr)
flag=True
for i in range(len(arr)):
    if arr[i]==key:
        flag=False
        print("element found at position ",i+1)
        break
if flag:
    print("element not found!")